src/System.ComponentModel.Composition.Registration/pkg/System.ComponentModel.Composition.Registration.pkgproj

```diff
@@ -2,7 +2,7 @@
     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
     <ItemGroup>
     <ProjectReference Include="..\ref\System.ComponentModel.Composition.Registration.csproj">
-      <SupportedFramework>uap10.0.16299;netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp3.0;net45</SupportedFramework>
```

Comment:
Curious: why'd you have to remove all Xamarin here?

Reply:
AllXamarinFrameworks aren't supported on netstandard2.1. I guess the right fix here would to mark it as inbox there?